<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>otls folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>archive folder</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="otls_treeWidget">
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>path</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTreeWidget" name="archive_treeWidget">
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>path</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="toArchiveButton">
     <property name="text">
      <string>Move to archive</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="toOtlsButton">
     <property name="text">
      <string>Move to otls</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
